### PR TITLE
stall hahn hack

### PR DESCRIPTION
**Pull request by jsteemann**

### CODE REVIEW

**Comment by jsteemann** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/4118/

File: arangod/RocksDBEngine/RocksDBKeyBounds.cpp [hunk 1]
```diff
@@ -315,11 +315,18 @@ RocksDBKeyBounds::RocksDBKeyBounds(RocksDBEntryType type, uint64_t first)
...
       uint64ToPersistent(_internals.buffer(), first);
       _internals.buffer().append((char*)(min.begin()), min.byteSize());
+      //uint64ToPersistent(_internals.buffer(), first - 1);
+      //_internals.buffer().append((char*)(max.begin()), max.byteSize());
 
       _internals.separate();
 
-      uint64ToPersistent(_internals.buffer(), first);
-      _internals.buffer().append((char*)(max.begin()), max.byteSize());
+      if (false && rocksDBEndianness == RocksDBEndianness::Big) {
```
Comment (goedderz) [inline arangod/RocksDBEngine/RocksDBKeyBounds.cpp]: Is this WIP?

Comment (jsteemann) [reply arangod/RocksDBEngine/RocksDBKeyBounds.cpp]: Yes, definitely!

**Comment by jsteemann** [comment]
Closing as obsolete

**Pull request closed without merging**